lib/mongo/client.rb

```diff
@@ -157,6 +158,8 @@ def hash
     # @option options [ Integer ] :server_selection_timeout The timeout in seconds
     #   for selecting a server for an operation.
     # @option options [ String ] :password The user's password.
+    # @option options [ Integer ] :max_idle_time The maximum seconds a socket can remain
+    #   idle in the connection pool after it has been checked out and checked back in.
```

Comment:
This is a little confusing, perhaps "maximum seconds a socket can remain idle since it has been checked in to the pool."